src/codeu/chat/server/Server.java

```diff
@@ -53,10 +54,17 @@
   private final Timeline timeline = new Timeline();
 
   private final Map<Integer, Command> commands = new HashMap<>();
-
   private final Uuid id;
   private final Secret secret;
+  private ServerInfo info = null;
 
+  {
+    try {
+      info = new ServerInfo(Uuid.parse("1.0.0"));
```

Comment:
I would add a note making sure people know to change this variable whenever they change the server code. Or maybe move the `String` into its own variable?

Actually now that I'm looking at it, I realize it would probably be better to just move this whole thing inside the `Server` constructor instead of using an initialization block. I'll leave the final decision up to you though.